src/components/Switch/Switch.tsx

```diff
@@ -1,9 +1,27 @@
-import React, { useMemo, useRef } from "react";
+import React, { FC, ReactElement, useMemo, useRef } from "react";
 import cx from "classnames";
-import { useSwitchChecked } from "../../components/Switch/hooks/useSwitchChecked";
+import { useSwitchChecked } from "./hooks/useSwitchChecked";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { MockToggleProps } from "../Toggle/MockToggle";
 import classes from "./Switch.module.scss";
 
-export const Switch = ({
+interface SwitchProps extends VibeComponentProps {
+  name?: string;
```

Comment:
[Optional]
It's not part of this PR but in future we have to add here descriptions of props (as in other Components)